misc/FFNx.cfg

```diff
@@ -63,59 +91,138 @@
-#show_fps = no
+#[USE EXTERNAL MUSIC]
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# This flag will enable/disable the support of an enhanced music layer to reproduce music in-game.
+# If you leave out the default configuration
+# FFNx will autodetect your environment and will set it to the best available option.
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#use_external_music = yes
 
-# show the chosen renderer backend in the title bar
-#show_renderer_backend = yes
+# [EXTERNAL MUSIC PATH]
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Path of the external music files
+# Will try to load from this path before using the default for your Version of the game
+# Defaults:
+# - FF7 1998: music/vgmstream ( 7h-era compatibility )
+# - FF7 eStore: data/music_ogg
+# - FF7 Steam: data/music_ogg
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#external_music_path =
+
+# [EXTERNAL MUSIC EXTENSION]
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# The type of file to search for. By default is ogg.
+# Supported extensions: 
+# - https://github.com/losnoco/vgmstream#supported-file-types
+# - Any additional or any type supported by your loaded winamp_in_plugin
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#external_music_ext = ogg
+
+
+# [EXTERNAL WINAMP PLUGIN SUPPORT]
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# These flags will allow you to make use of compatible in_* and out_* DLL plugins you were used to load on Winamp itself.
+# Through these flags you can for example decide to use an input plugin to play something FFNx never though about ( like MINIPSF files )
+# and output the sound data to your custom plugin which may add custom reverb and much more.
+# By default winamp_in_plugin will use the built-in VGMStream layer.
+# By default winamp_out_plugin will use the built-in output layer.
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#winamp_in_plugin =
+#winamp_out_plugin =
 
-## INTERNALS
+###########################
+# Video Player Options
+###########################
 
-# external video
+#[ENABLE FFMPEG VIDEOS]
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
 # This flag will enable/disable the support of FFMpeg layer to reproduce movies in-game.
-# By default this flag is enabled for FF7 1998, FF7/FF8 Steam edition and disabled for FF8 2000 edition.
+# Default Value depends on game version.
+# - FF7 1998 - yes
+# - FF7/FF8 Steam - yes
+# - FF8 2000 - no
 # It is suggested to keep the default behavior unless you really know what are you doing.
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
 #enable_ffmpeg_videos = yes
 
-# external video extension
-# This flag will allow you to customize the video extension to be used. By default is avi.
-#external_movie_ext = avi
+#[FFMPEG VIDEO FILE EXTENSION]
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# The type of file that the ffmpeg layer will search for. Default is avi.
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#ffmpeg_video_ext = avi
 
-# STEAM ONLY!
+
+###############################
+# MUST SET FOR VERSIONS BELOW
+# FF7 2012   FF7 STEAM
+# FF8 2013   FF8 STEAM

```

Comment:
FF8 2013 and FF8 Steam isn't the game edition? Or am I missing something?

Reply:
No the 2013 release is done by square enix  before a steam release but idk that they still sell it . much like the 2012 release of FF7 it seams to have bee discontinued in place of the Steam realease

Reply:
Good to know :) So there's an eStore release of FF8 then...guess I need to update the README too. Thanks!